engine/class_modules/paladin/sc_paladin.cpp

```diff
@@ -2524,6 +2524,23 @@ void paladin_t::init_action_list()
   player_t::init_action_list();
 }
 
+// paladin_t::validate_fight_style ==========================================
+bool paladin_t::validate_fight_style( fight_style_e style ) const
+{
+  if ( specialization() == PALADIN_PROTECTION )
+  {
+    switch ( style )
+    {
+      case FIGHT_STYLE_DUNGEON_ROUTE:
+      case FIGHT_STYLE_DUNGEON_SLICE:
+        return true;
+      default:
+        return false;
+    }
+  }
+  return true;
+}
+
```

Comment:
```suggestion
bool paladin_t::validate_fight_style( fight_style_e style ) const
{
  if ( specialization() == PALADIN_PROTECTION )
  {
    switch ( style )
    {
      case FIGHT_STYLE_DUNGEON_ROUTE:
      case FIGHT_STYLE_DUNGEON_SLICE:
        return false;
      default:
        return true;
    }
  }
  return true;
}

```

Reply:
Manually corrected for in conjunction with combining changes from #8665.